.travis.yml

```diff
@@ -20,11 +17,14 @@ matrix:
     - os: linux
       dist: trusty
       sudo: required
-      env: SWIFT_SNAPSHOT=4.0-DEVELOPMENT-SNAPSHOT-2017-06-11-a
+      env: SWIFT_SNAPSHOT=4.0-DEVELOPMENT-SNAPSHOT-2017-07-05-a
```

Comment:
Is there an advantage to standardizing to one snapshot over just using the latest snapshot for whatever repo we are doing that day?